index.html

```diff
@@ -76,7 +76,7 @@ <h3>Tech stack</h3>
             src="https://upload.wikimedia.org/wikipedia/commons/c/c3/Python-logo-notext.svg" width="32" height="32"
             alt="python-logo"></a>
         <!-- Kotlin-logo with a link to kotlinlang.org -->
-        <a id="kotlin" class="logo" href="https://kotlinlang.org/" target="_blank"><img
+        <a id="kotlin-logo" class="logo" href="https://kotlinlang.org/" target="_blank"><img
```

Comment:
Cool do you write your own apps for Android?

Reply:
I tried 2 years ago, with some android and kotlin books and the fantastic android studio software. But i think it is a very difficult start into the world of development. I retry it with more experience.